AutomaticClockOuts/constants.js

```diff
@@ -0,0 +1,6 @@
+const constants = {
+  ClockOutMessage : 'OMG, you have been working more than 12 hours in a row. \nPlease take a break and visit https://timetracker.ioet.com/ to set the right end time for your entries, we just did a clock out for you :wink:',
+  time_tracker_channel: 'C01694949JR'
```

Comment:
I think you need to hide this ID and use the .env to get this var, very similar to SLACK-TOKEN. @diegotony  can help to add this new var for example `ID-CHANNEL-NOTIFY`

Reply:
In my opinion the channel id is not necessary to hide it in the .env, when you enter slack from the browser you can get it from the URL. The permissions are given in the bot and it handles them with the actual variable `SLACK_TOKEN_NOTIFY` but if you think is necessary it can handle as variable on .env.

Reply:
> when you enter slack from the browser you can get it from the URL.
I am not sure if hide or not. You guys can give your better arguments.

But I think we should make this values as CAPITAL Letters. `CLOCK_OUT_MESSAGE`, `TIME_TRACKER_CHANNEL_ID`.

Reply:
@Angeluz-07 you are right , I am going to use capital Letters, for me is similar than ClientID here https://api.slack.com/authentication/best-practices comment that you can `Your Client ID can be shared freely in code and email ...`